### PR TITLE
Fixes #17217: Replace node-to-relay script from rudder-package

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/17217

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
This script should only be in the plugin now

File: rudder-webapp/SOURCES/rudder-node-to-relay [hunk 1]
```diff
@@ -25,233 +23,22 @@ set -e
...
-EXISTING_NODEID=$(${LDAPSEARCH} -s base -b "nodeGroupId=hasPolicyServer-${RELAY_UUID},groupCategoryId=SystemGroups,groupCategoryId=GroupRoot,ou=Rudder,cn=rudder-configuration" nodeId 2>/dev/null | grep "^nodeId:" || true)
-EXISTING_NETWORKS=$(${LDAPSEARCH} -s base -b "directiveId=common-root,activeTechniqueId=common,techniqueCategoryId=Rudder Internal,techniqueCategoryId=Active Techniques,ou=Rudder,cn=rudder-configuration" directiveVariable 2>/dev/null | grep "^directiveVariable: ALLOWEDNETWORK\[" || true)
-EXISTING_TYPE=$(${LDAPSEARCH} -s base -b "nodeId=${RELAY_UUID},ou=Nodes,cn=rudder-configuration" isSystem 2>/dev/null | grep "^isSystem:" || true)
-RELAY_HOSTNAME=$(${LDAPSEARCH} -s base -b "nodeId=${RELAY_UUID},ou=Nodes,ou=Accepted Inventories,ou=Inventories,cn=rudder-configuration" nodeHostname 2>/dev/null | grep "^nodeHostname:" | sed 's%nodeHostname: %%' || true)
-
-if [ "${RELAY_HOSTNAME}" = "" ]; then
-	${ECHO} "ERROR: Cannot find the machine (inventory not found), aborting."
-	exit 1
-fi
+echo "INFO: Please use 'rudder server node-to-relay <UUID>' command"
```
Comment (amousset) [inline rudder-webapp/SOURCES/rudder-node-to-relay:37]: This command is deprecated, please ...

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
Commit modified

File: rudder-webapp/SOURCES/rudder-node-to-relay [hunk 2]
```diff
@@ -25,233 +23,22 @@ set -e
...
-RELAY_HOSTNAME=$(${LDAPSEARCH} -s base -b "nodeId=${RELAY_UUID},ou=Nodes,ou=Accepted Inventories,ou=Inventories,cn=rudder-configuration" nodeHostname 2>/dev/null | grep "^nodeHostname:" | sed 's%nodeHostname: %%' || true)
-
-if [ "${RELAY_HOSTNAME}" = "" ]; then
-	${ECHO} "ERROR: Cannot find the machine (inventory not found), aborting."
-	exit 1
-fi
+echo "INFO: This command is deprecated, please use 'rudder server node-to-relay <UUID>'"
+echo -e "Running: rudder server node-to-relay $@ \n"
+rudder server node-to-relay $@
+code=$?
```
Comment (amousset) [inline rudder-webapp/SOURCES/rudder-node-to-relay:40]: no need for check here, let the new script do the error handling

**Comment by ElaadF** [comment]
PR updated with a new commit

**Comment by ElaadF** [comment]
OK, squash merging this PR

**Pull request merged**